### ISSUE TITLE
Change prefix for configuration properties from `quarkus.operator-sdk` to `qosdk`?

### ISSUE THREAD

**Issue by metacosm** [quarkiverse/quarkus-operator-sdk#733]
This would be shorter to type and would also bypass conversion issues between env variables and property names.

**Comment by kdubb** [comment]
This seems like a pretty big departure from Quarkus norms for a Quarkiverse project (which all live under `quarkus.`), for a few keystrokes.

Just dropping the `-sdk`. would get you nearly the same savings; and even then I don't think it's worth the deviation.

**Comment by metacosm** [comment]
> This seems like a pretty big departure from Quarkus norms for a Quarkiverse project (which all live under `quarkus.`), for a few keystrokes.

Indeed, but then again, I think that having everything with the same prefix makes it useless to have the prefix because it becomes just noise. So I feel that switching to `qosdk` as the prefix would streamline things and make it easier for folks to discover and type properties (even when there's no auto-completion available). 

> 
> Just dropping the `-sdk`. would get you nearly the same savings; and even then I don't think it's worth the deviation.

Dropping the `-sdk` part is needed because this will remove ambiguity during the env <-> property name conversion process which has and is causing issues.

**Comment by metacosm** [comment]
> > make it easier for folks to discover and type properties (even when there's no auto-completion available).
> 
> 
> 
> Objectively this will make it harder. Right now I know, without looking anywhere, it's the plugin name... so starts `quarks.operator-sdk.` or `QUARKUS_OPERATOR_SDK_`. Migrating from this will require me to look it up.

An example of the problem is how does Quarkus knows that `QUARKUS_OPERATOR_SDK_` maps to a property named `quarkus.operator-sdk` or `quarkus.operator.sdk`? 

Don't get me wrong, I get your points but there's a real issue at the moment with the prefix.


> 
> Maybe just alias them to some other root?
